spec/position_initializer_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'spec_helper'
+
+connect_to_sqlite
+
+describe RailsCoreExtensions::PositionInitializer, 'When repositioning' do
+  class Child < ActiveRecord::Base; end
+
+  before do
+    Child.create!(parent_id: 1, name: 'A child')
+  end
+
+  let(:child)  { Child.find_by_name('A child') }
+
+  context 'when not initialized' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.